docs/architecture/metadata/workflow.md

```diff
@@ -0,0 +1,30 @@
+## Live Reloading of a Data Page
```

Comment:
It looks like it only covers data-page specific use case? If so, I'd change the entry name from Workflow to something else, and level up the headings.

Or if you think this should cover other use cases in the future, I'd clarify at the top of the page that we only talk about data-page but will include other types of workflows in the future.

Reply:
Good point. Fixed.